README.rst

```diff
@@ -0,0 +1,63 @@
+|Tests Status| |Coverage|
+
+`aehmc` provides MCMC sampling algorithms written in `Aesara <https://github.com/pymc-devs/aesara>`_.
+
+Features
+========
+- Sample from an (unnormalized) probability distribution using Hamiltonian Monte
+  Carlo and the No U-Turn Sampler.
+
+Example
+=======
+
+.. code-block:: python
+
+  import aesara
+  from aesara import tensor as at
+  from aesara.tensor.random.utils import RandomStream
+
+  from aeppl import joint_logprob
+
+  from aehmc import hmc as hmc
+
+  # A simple normal distribution
+  Y_rv = at.random.normal(0, 1)
+
+  def logprob_fn(y):
+      logprob = joint_logprob(Y_rv, {Y_rv: y})
+      return logprob
+
+  # Build the transition kernel
+  srng = RandomStream(seed=0)
+  kernel = hmc.kernel(
```

Comment:
This makes me think we should rename `hmc.kernel` to something like `hmc.[create_]kernel_function`.

Reply:
I've been wondering that too. I'll open a new issue.